dev/com.ibm.ws.jaxws.2.3.webcontainer/bnd.bnd

```diff
@@ -0,0 +1,80 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+bVersion=1.0
+
+Bundle-Name: JAX-WS WebContainer
+Bundle-SymbolicName: com.ibm.ws.jaxws.2.3.webcontainer
+Bundle-Description: IBM JAX-WS 2.3 Web Container; version=${bVersion}
+
+Service-Component: com.ibm.ws.jaxws.webcontainer.jaxWsWebContainerManager; \
+     implementation:=com.ibm.ws.jaxws23.webcontainer.JaxWsWebContainerManagerImpl; \
+     provide:=com.ibm.ws.jaxws.support.JaxWsWebContainerManager; \
+     webModuleHandler='com.ibm.ws.app.manager.module.internal.ModuleHandler(type=web)'; \
+     artifactContainerFactory=com.ibm.wsspi.artifact.factory.ArtifactContainerFactory; \
+     adaptableModuleFactory= com.ibm.wsspi.adaptable.module.AdaptableModuleFactory; \
+     wsLocationAdmin=com.ibm.wsspi.kernel.service.location.WsLocationAdmin; \
+     properties:="service.vendor=IBM", \
+  com.ibm.ws.jaxws.webcontainer.extensionFactory; \
+     implementation:=com.ibm.ws.jaxws23.webcontainer.JaxWsExtensionFactory; \
+     provide:='com.ibm.wsspi.webcontainer.extension.ExtensionFactory'; \
+     endpointPublisherManager=com.ibm.ws.jaxws.endpoint.EndpointPublisherManager; \
+     jaxWsWebAppConfigurator=com.ibm.ws.jaxws.webcontainer.JaxWsWebAppConfigurator*; \
+     properties:="service.vendor=IBM", \
+  com.ibm.ws.jaxws.webcontainer.webEndpointPublisher; \
+     implementation:=com.ibm.ws.jaxws23.webcontainer.WebEndpointPublisher; \
+     provide:='com.ibm.ws.jaxws.endpoint.EndpointPublisher'; \
+     jaxWsEndpointConfiguratorManager=com.ibm.ws.jaxws.endpoint.JaxWsEndpointConfiguratorManager;\
+     ImplBeanCustomizer/registerImplBeanCustomizer=com.ibm.ws.jaxws.ImplBeanCustomizer*;\
+     properties:="service.vendor=IBM", \
+  com.ibm.ws.jaxws.router.initializer; \
+     implementation:=com.ibm.ws.jaxws23.webcontainer.JaxWsRouterServletContainerInitializer; \
+     provide:=javax.servlet.ServletContainerInitializer; \
+     immediate:=true;\
+     properties:="service.vendor=IBM"\
+  
+
+Export-Package: com.ibm.ws.jaxws23.webcontainer
+
+Import-Package: \
+   javax.xml.ws.*;version="[2.2,3)", \
+   org.apache.cxf.*;version="[2.6.2,2.6.3)", \
```

Comment:
This is neither here nor there but I'm not sure if you changed anything, I just  find it strange there is an import for org.apache.cxf.* but no CXF jars listed in the buildpath